pkg/image/digest/digest.go

```diff
@@ -0,0 +1,138 @@
+package digest
```

Comment:
this used to be in an `internal` package that prevented anyone from depending on it. We did this because it was a copy of existing code from elsewhere.